.github/workflows/docker.yml

```diff
@@ -50,9 +47,10 @@ jobs:
         with:
           name: artifacts
           path: dist
-      - name: npm install
+      - name: install dependencies
         run: npm ci
         env:
+          GH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
adjusted all actions to use the `GH_TOKEN` env-variable

Reply:
Nice 🔥